src/lib/openjp2/t1.c

```diff
@@ -1003,130 +1197,186 @@ static void opj_t1_enc_clnpass(
 	}
 }
 
-static void opj_t1_dec_clnpass(
+#define MACRO_t1_flags_internal(x,y,flags_stride) t1->flags[((x)*(flags_stride))+(y)]
+
+#define opj_t1_dec_clnpass_internal(consistency_check, t1, bpno, cblksty, w, h, flags_stride) \
```

Comment:
Possibly, but the resulting assembly code should be checked to verify that the compiler really inlines when w, h are set to 64